util/predictoor/query.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict
+
+from util.graphutil import submitQuery
+from util.predictoor.models import Predictoor, Prediction
+
+
+def queryPredictoors(st_block: int, end_block: int, chainID: int):
```

Comment:
please use snake case for functions

Reply:
yes, true, I want to discuss this further because it seems we have a mixture of patterns. Previously we did not have a lot of classes, so it was a convention-like thing. But as this repo/project grows we might think about implementing casing standards.